src/box/lua/info.c

```diff
@@ -741,6 +741,8 @@ lbox_info_synchro(struct lua_State *L)
 		lua_pushnumber(L, now - oldest_entry->insertion_time);
```

Comment:
Nit: in doc request you say, that the feature is from 3.1. But the version was already released and the feature will be only in 3.2. Or we're going to backport it to 3.1 (which I don't like)

Reply:
Fixed.